frontend/src/components/Options.vue

```diff
@@ -172,6 +296,22 @@ export default Vue.extend({
     canClaimTokens(): boolean {
       return !toBN(this.skillRewards).lte(0);
     },
+    supportedProjects(): SupportedProject[] {
+      const supportedProjects = this.getPartnerProjects.map(p => {
+        return {
+          id: p.id,
+          name: p.name,
+          tokenSymbol: p.tokenSymbol,
+          tokenAddress: p.tokenAddress,
+          tokenSupply: p.tokenSupply,
+          tokensClaimed: p.tokensClaimed,
+          tokenPrice: p.tokenPrice,
+          isActive: p.isActive
```

Comment:
What is this mapping for? Am I missing something or are you mapping all the fields to the same fields?